src/Grafana/dashboards/aspnetcore.json

```diff
@@ -1329,9 +1342,10 @@
     ]
   },
   "time": {
-    "from": "now-5m",
+    "from": "now-24h",
```

Comment:
Changed the default time horizon to 24h - I can change it back to 5m. Just happened to be set that way when I exported it.